### ISSUE TITLE
Allelic requirement

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7492]
**Preferred term label:**
Allelic requirement

**Synonyms**
Mode of inheritance HP:0000005

**Definition (free text, please give PubMed ID)**
While there is a close conceptual relationship between allelic requirement and inheritance, they
serve different purposes; allelic requirement is necessary for variant annotation
pipelines and to determine if a given variant in a specific context is relevant to the phenotype of the
patient e.g. a heterozygous variant (in the absence of compound heterozygosity) is relevant for dominant
but not for recessive disease; while inheritance is used for describing the mode of inheritance of a phenotype and
is particularly applicable in the clinical setting for communicating recurrence risk, to guide family
screening, and to guide reproductive advice.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
While I understand this general use case, variant annotation pipelines are not in the scope of the HPO. Would it be OK for your use case if we added the terms here as synonyms to the Mode of inheritance terms rather than creating a new subontology?

**Comment by pnrobinson** [comment]
@angharadroberts Have you had a chance to think about this? I think it is best to add them as synonyms, I will do so unless there is a reason why this would not work?

**Comment by angharadroberts** [comment]
@pnrobinson I had replied to your other email/comment about this.  We had envisioned allelic requirement as a new subontology that linked to relevant inheritance terms.  At the time of our call in Nov 2020 to review our set of terms and how we might fit these into the HPO framework, we had 2 separate sets of terms: inheritance and allelic requirement.  I recall you questioned if these needed to be separate entities.  This was then discussed in detail on the following GenCC call and the group, including Ada, felt it was important to keep both. Although each allelic requirement term has an inheritance term that it maps to, there is a conceptual difference between inheritance and allelic requirement; many HPO inheritance terms would not map directly to proposed allelic requirement term; and I think having a separately subontology would make clear the conceptual difference but would it easier to navigate allelic requirement terms.  
I’d be heppt to discuss futher by email or to set up a call if either would be helpful.
Kind regards, Ang

**Comment by pnrobinson** [comment]
Hi @angharadroberts 
Why don't we zoom and try to figure out the best way forward? Although I understand the conceptual difference, I am not sure I understand the use case. In the end, we are responsive to the community, and will make the terms if needed, but I would like to try to convince you in a friendly debate that they are not needed, if you understand what I mean.
I am peter dot robinson at jax . org

**Comment by pnrobinson** [comment]
I have added the above text as a comment to the [Mode of inheritance HP:0000005](https://hpo.jax.org/app/browse/term/HP:0000005) term

**Comment by angharadroberts** [comment]
Dear Peter,
Sorry for the delay in getting back to you.
We had envisioned allelic requirement as a new subontology that linked to relevant inheritance terms.  At the time of our call in Nov 2020 to review our set of terms and how we might fit these into the HPO framework, we had 2 separate sets of terms: inheritance and allelic requirement.  I recall you questioned if these needed to be separate entities.  This was then discussed in detail on the following GenCC call and the group, including Ada, felt it was important to keep both. Although each allelic requirement term has an inheritance term that it maps to, there is a conceptual difference between inheritance and allelic requirement; many HPO inheritance terms would not map directly to proposed allelic requirement term; and I think having a separately subontology would make clear the conceptual difference but would it easier to navigate allelic requirement terms.
I’d be heppt to discuss futher by email or to set up a call if either would be helpful.
Kind regards
Ang


Angharad Roberts
Clinical Research Fellow
Cardiovascular Genetics and Genomics
National Heart and Lung Institute
Imperial College London
email: ***@***.******@***.***>

www.cvgenetics.org<https://gbr01.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.cvgenetics.org%2F&data=04%7C01%7CCarp%40mrc.ukri.org%7C0548425ae69f45e09ff408d9a44ecb90%7C8bb7e08edaa44a8e927efca38db04b7e%7C0%7C0%7C637721481555497905%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=JrBVKfSXTdNC3GaO3LZaHlBFirBVxZULd2DKRg7oPgw%3D&reserved=0>


[A picture containing logo  Description automatically generated]




From: Peter Robinson ***@***.***>
Date: Wednesday, 11 May 2022 at 14:12
To: obophenotype/human-phenotype-ontology ***@***.***>
Cc: Roberts, Angharad M ***@***.***>, Author ***@***.***>
Subject: Re: [obophenotype/human-phenotype-ontology] Allelic requirement (Issue #7492)
This email from ***@***.*** originates from outside Imperial. Do not click on links and attachments unless you recognise the sender. If you trust the sender, add them to your safe senders list<https://spam.ic.ac.uk/SpamConsole/Senders.aspx> to disable email stamping for this address.



While I understand this general use case, variant annotation pipelines are not in the scope of the HPO. Would it be OK for your use case if we added the terms here as synonyms to the Mode of inheritance terms rather than creating a new subontology?
